Cargo.lock

```diff
@@ -1034,8 +1034,7 @@ dependencies = [
 [[package]]
 name = "ibc"
 version = "0.51.0"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "8af2325614ae3274b2e70834bea56c1fa2758d802d364d89992b0fe820847fca"
+source = "git+https://github.com/cosmos/ibc-rs.git#f7a3c2b23f42977392b2f2ca908cc98d00765ff5"
```

Comment:
looks like, the rev is set to wrong rev (it's the latest commit on `main`) :sweat_smile:

Reply:
Whoops. I switch a lot between `main` and `feat/sov-support`, that's probably why!